web/src/pages/settings/ManageLibrariesModal.tsx

```diff
@@ -39,8 +41,30 @@ const CreateLibraryForm: React.FC<{ onDone: (id: number) => void }> = ({ onDone
         submitLabel="Create"
       >
         <TextInput required label="Name:" value={name} onChange={setName} id="name" />
-        <TextInput required label="Folder:" value={folder} onChange={setFolder} id="name" />
-        <SelectField options={subFolders} onSelect={setFolder} id="subfolders" />
+        <TextInput
+          required
+          label={
+            <>
+              <IconButton
+                className="parent-dir"
+                type="button"
+                disabled={!browseSet || browseSet.parent === browseSet.path}
```

Comment:
Currently the up button is not disabled when the "Folder" input is empty, but we _are_ looking at the top-most directory (i.e. just `/` on unix):

![image](https://user-images.githubusercontent.com/5592716/125824066-71bdb0ac-1d39-4bf8-80aa-657cce599da4.png)

Pressing it sets folder to a single dot (`.`). Nothing seems to break, but I'd expect the button to be disabled.

Reply:
Fair